clippings-parser-2.py

```diff
@@ -1,10 +1,11 @@
+import re
 from typing import List
 from clippingsParserHelpers import *
-
+import os
 # Location of clippings file
 # TODO: import this with a YAML file later
-CLIPPINGS = 'My Clippings.txt'
-
+CLIPPINGS = r"C:\Users\vinee\OneDrive\Kindle Screenshots\My Clippings\My Clippings 2021_02_25.txt"
```

Comment:
The path to the clippings text file cannot be hard-coded like this to cater to a single user. It needs to be generic so it serves all.
Presently, by making the path "My Clippings.txt", I'm assuming the user would put the file on the root folder of this repo. This may not be ideal, and better solutions would be:
- to get the path through CLI params from the user
- to make a JSON / YAML file for the settings, and get the path from there.

I would welcome any of the above changes if implemented well, but not the current hardcoded one. So for now, I'm closing this PR.